src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final DateTimeFormatter sdf = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate startDate = LocalDate.parse(dateFrom, sdf);
+        LocalDate stopDate = LocalDate.parse(dateTo, sdf);
+        String[] splitData;
+        LocalDate date;
+        String dataName;
+        int hours;
+        int wage;
+        int salary;
```

Comment:
let's move these variables closer to the place of first use. Also remove redundant ones